src/gui/src/gui.cpp

```diff
@@ -562,17 +562,24 @@ void Gui::showGui(const std::string& cmds, bool interactive)
     return;
   }
 
+  int argc = 1;
+  char* argv = new char[9];
+  strcpy(argv, "openroad");
```

Comment:
shouldn't we pass in the real argc/argv?  "This has the side benefit of allowing commandline arguments that Qt can use to be passed in when using -gui"

Reply:
I suppose we could grab that from main.cc since it's static (but once they have been consumed the arguments are removed from argv). The real argc/argv are used in main.cc.
This code is used to open the gui again, and therefore it is not from the command line.